README.md

```diff
@@ -1 +1,5 @@
 # Peer Programming Challanges
+Use this project as a way to challange developers with relativly real world examples related to RESTful APIs and frontend technologies. By loading this repository via [https://vscode.dev/github/heyitsgeo/peer-programming-challanges](https://vscode.dev/github/heyitsgeo/peer-programming-challanges) developers can edit code in a browser without needing to install or clone anything on their machine.
```

Comment:
Use this project to challenge developers with relatively real-world examples related to RESTful APIs and frontend technologies. By loading this repository via [https://vscode.dev/github/heyitsgeo/peer-programming-challanges](https://vscode.dev/github/heyitsgeo/peer-programming-challanges) developers can edit code in a browser without needing to install or clone anything on their machine.